packages/mux-player/src/styles.css

```diff
@@ -22,21 +21,24 @@ a {
   cursor: pointer;
 }
 
-[part~='seek-live'][aria-disabled] {
-  cursor: default;
-  pointer-events: none;
-}
-
 [part~='seek-live']::before {
   content: '● ';
   color: rgb(118, 118, 118);
   padding-right: 5px;
 }
-
-[part~='seek-live'][aria-disabled]::before {
+[part~='seek-live'][in-live-window]::before {
   color: #fb3c4d;
 }
 
+[part~='seek-live'][disabled] {
+  opacity: 60%;
+  cursor: not-allowed;
+}
+[part~='seek-live'][in-live-window] {
+  opacity: 100%;
+  cursor: unset;
```

Comment:
if we're in-live-window, the control is also disabled, which gets a 60% opacity, but we want it visible to users here and not dimmed.
Same with the cursor, we want it to not be a pointer but not be not-allowed.

Reply:
good description.